cmd/minikube/cmd/node_add.go

```diff
@@ -36,13 +41,46 @@ var (
 	cpNode              bool
 	workerNode          bool
 	deleteNodeOnFailure bool
+	osType              string
+	// windowsVersion      string
+
+	osTypeLong = templates.LongDesc(i18n.T(`
+		This flag should only be used when adding a windows node to a cluster.
```

Comment:
Only?

Reply:
At the moment we are only supporting this feature on Windows node hence this flag!